lib/service/logger.js

```diff
@@ -24,7 +24,7 @@ function _log(level, message) {
     level
   };
 
-  logMessage = JSON.stringify(logMessage) + '\n';
+  logMessage = `${JSON.stringify(logMessage)}\n`;
```

Comment:
@mjsalinger I wonder if there's any difference in this case. `JSON.stringify` already returns a string, so it seems to me that `returnAString() + 'newString'` should create the same number of objects as `` `${returnAString()}newString` ``